### ISSUE TITLE
Error: Can't find stylesheet to import (Payload 2.0)

### ISSUE THREAD

**Issue by silveltman** [DennisSnijder/payload-workflow#2]
Awesome looking plugin!

Just wanted to try it out but get the error below. I'm updated to Payload 2.0, so maybe that has something to do with it?

```
silveltman@Sils-MBP payload % yarn dev
yarn run v1.22.19
$ cross-env PAYLOAD_CONFIG_PATH=src/payload.config.ts nodemon
[nodemon] 3.0.1
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: ts
[nodemon] starting `ts-node src/server.ts`
[19:54:07] INFO (payload): Connected to MongoDB server successfully!
[19:54:07] INFO (payload): Starting Payload...
[19:54:07] INFO (payload): Payload Admin URL: http://localhost:3000/admin
[19:54:07] INFO (payload): App URL: http://localhost:3000
[sass] Can't find stylesheet to import.
  ╷
1 │ @import "payload/scss/index";
  │         ^^^^^^^^^^^^^^^^^^^^
  ╵
  node_modules/payload-workflow/dist/components/WorkflowViewHeader/styles.scss 1:9  root stylesheet
[sass] Can't find stylesheet to import.
  ╷
1 │ @import "payload/scss/index";
  │         ^^^^^^^^^^^^^^^^^^^^
  ╵
  node_modules/payload-workflow/dist/components/BoardCard/sytles.scss 1:9  root stylesheet
[sass] Can't find stylesheet to import.
  ╷
1 │ @import "payload/scss/index";
  │         ^^^^^^^^^^^^^^^^^^^^
  ╵
  node_modules/payload-workflow/dist/components/BoardColumn/styles.scss 1:9  root stylesheet
[sass] Can't find stylesheet to import.
  ╷
1 │ @import "payload/scss/index";
  │         ^^^^^^^^^^^^^^^^^^^^
  ╵
  node_modules/payload-workflow/dist/components/WorkflowView/styles.scss 1:9  root stylesheet
9:54:23 PM [VITE-WARNING] Internal server error: [sass] Can't find stylesheet to import.
  ╷
1 │ @import "payload/scss/index";
  │         ^^^^^^^^^^^^^^^^^^^^
  ╵
  node_modules/payload-workflow/dist/components/WorkflowViewHeader/styles.scss 1:9  root stylesheet
  Plugin: vite:css
  File: /Users/silveltman/Documents/Github/monorepo/payload/node_modules/payload-workflow/dist/components/WorkflowViewHeader/styles.scss:1:9
  Error: Can't find stylesheet to import.
    ╷
  1 │ @import "payload/scss/index";
    │         ^^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules/payload-workflow/dist/components/WorkflowViewHeader/styles.scss 1:9  root stylesheet
      at Object.wrapException (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:1269:17)
      at Object.throwWithTrace0 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:24245:15)
      at /Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:73332:19
      at _wrapJsFunctionForAsync_closure.$protected (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:3939:15)
      at _wrapJsFunctionForAsync_closure.call$2 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:29731:12)
      at _awaitOnObject_closure.call$1 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:29719:32)
      at Object._rootRunUnary (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:4313:18)
      at StaticClosure.<anonymous> (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:103106:16)
      at _CustomZone.runUnary$2$2 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:31127:39)
      at _Future__propagateToListeners_handleValueCallback.call$0 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:30200:51)
9:54:23 PM [VITE-WARNING] Internal server error: [sass] Can't find stylesheet to import.
  ╷
1 │ @import "payload/scss/index";
  │         ^^^^^^^^^^^^^^^^^^^^
  ╵
  node_modules/payload-workflow/dist/components/BoardCard/sytles.scss 1:9  root stylesheet
  Plugin: vite:css
  File: /Users/silveltman/Documents/Github/monorepo/payload/node_modules/payload-workflow/dist/components/BoardCard/sytles.scss:1:9
  Error: Can't find stylesheet to import.
    ╷
  1 │ @import "payload/scss/index";
    │         ^^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules/payload-workflow/dist/components/BoardCard/sytles.scss 1:9  root stylesheet
      at Object.wrapException (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:1269:17)
      at Object.throwWithTrace0 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:24245:15)
      at /Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:73332:19
      at _wrapJsFunctionForAsync_closure.$protected (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:3939:15)
      at _wrapJsFunctionForAsync_closure.call$2 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:29731:12)
      at _awaitOnObject_closure.call$1 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:29719:32)
      at Object._rootRunUnary (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:4313:18)
      at StaticClosure.<anonymous> (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:103106:16)
      at _CustomZone.runUnary$2$2 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:31127:39)
      at _Future__propagateToListeners_handleValueCallback.call$0 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:30200:51)
9:54:23 PM [VITE-WARNING] Internal server error: [sass] Can't find stylesheet to import.
  ╷
1 │ @import "payload/scss/index";
  │         ^^^^^^^^^^^^^^^^^^^^
  ╵
  node_modules/payload-workflow/dist/components/BoardColumn/styles.scss 1:9  root stylesheet
  Plugin: vite:css
  File: /Users/silveltman/Documents/Github/monorepo/payload/node_modules/payload-workflow/dist/components/BoardColumn/styles.scss:1:9
  Error: Can't find stylesheet to import.
    ╷
  1 │ @import "payload/scss/index";
    │         ^^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules/payload-workflow/dist/components/BoardColumn/styles.scss 1:9  root stylesheet
      at Object.wrapException (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:1269:17)
      at Object.throwWithTrace0 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:24245:15)
      at /Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:73332:19
      at _wrapJsFunctionForAsync_closure.$protected (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:3939:15)
      at _wrapJsFunctionForAsync_closure.call$2 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:29731:12)
      at _awaitOnObject_closure.call$1 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:29719:32)
      at Object._rootRunUnary (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:4313:18)
      at StaticClosure.<anonymous> (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:103106:16)
      at _CustomZone.runUnary$2$2 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:31127:39)
      at _Future__propagateToListeners_handleValueCallback.call$0 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:30200:51)
9:54:23 PM [VITE-WARNING] Internal server error: [sass] Can't find stylesheet to import.
  ╷
1 │ @import "payload/scss/index";
  │         ^^^^^^^^^^^^^^^^^^^^
  ╵
  node_modules/payload-workflow/dist/components/WorkflowView/styles.scss 1:9  root stylesheet
  Plugin: vite:css
  File: /Users/silveltman/Documents/Github/monorepo/payload/node_modules/payload-workflow/dist/components/WorkflowView/styles.scss:1:9
  Error: Can't find stylesheet to import.
    ╷
  1 │ @import "payload/scss/index";
    │         ^^^^^^^^^^^^^^^^^^^^
    ╵
    node_modules/payload-workflow/dist/components/WorkflowView/styles.scss 1:9  root stylesheet
      at Object.wrapException (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:1269:17)
      at Object.throwWithTrace0 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:24245:15)
      at /Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:73332:19
      at _wrapJsFunctionForAsync_closure.$protected (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:3939:15)
      at _wrapJsFunctionForAsync_closure.call$2 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:29731:12)
      at _awaitOnObject_closure.call$1 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:29719:32)
      at Object._rootRunUnary (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:4313:18)
      at StaticClosure.<anonymous> (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:103106:16)
      at _CustomZone.runUnary$2$2 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:31127:39)
      at _Future__propagateToListeners_handleValueCallback.call$0 (/Users/silveltman/Documents/Github/monorepo/payload/node_modules/sass/sass.dart.js:30200:51)
```

**Comment by DennisSnijder** [comment]
@silveltman thanks for the report! I'm working on 2.0 compatibility later this week! I'll be sure to fix this as well 😄

**Comment by DennisSnijder** [comment]
@silveltman I just released `payload-workflow@0.3.0` 😄, which supports Payload 2.0 going forward!

**Comment by silveltman** [comment]
Awesome! Error is fixed on my end 👌
